Add Stefan's Rust interview (#2131)
mainmatter/mainmatter.com: src/posts/2023-09-27-exploring-rusts-impact-on-costs-and-efficiency.md

---
title:
  "Exploring Rust's impact on efficiency and cost-savings - with Stefan
  Baumgartner"
authorHandle: marcoow
tags: [rust]
bio: "Marco Otte-Witte"
description:

>   "Rob Ede and Marco Otte-Witte discuss the impact of Rust on efficiency and

Reviewer: ```suggestion
  "Stefan Baumgartner and Marco Otte-Witte discuss the impact of Rust on efficiency and
```

  operating costs and Stefan's experience learning and teaching Rust."
og:
  image: /assets/images/posts/2023-09-27-exploring-rusts-impact-on-costs-and-efficiency/og-image.png
tagline: |
  <p>Senior Product Architect at Dynatrace Stefan Baumgartner and Mainmatter’s Founder Marco Otte-Witte discuss the benefits and use cases of Rust adoption and Stefan shares his experience learning and teaching Rust first-hand.</p>

image: "/assets/images/posts/2023-09-27-exploring-rusts-impact-on-costs-and-efficiency/header-visual.jpg"
imageAlt: "Stefan's picture on a gray background"
---

According to Marco Otte-Witte, Founder of Mainmatter, and Stefan Baumgartner,
Senior Product Architect at Dynatrace, Rust allows developers to write more
stable software with fewer maintenance issues and higher efficiency, resulting
in cost savings for companies. In the interview, Stefan shared his experience
introducing Rust to Dynatrace’s platform, and how they delivered a functioning

> prototype within a few days for a particular use case. Ever since, the small

Reviewer: ```suggestion
prototype within a few days for a particular use case. Ever since, his
```

team at Dynatrace has been able to maintain and run the Rust-based product
smoothly with minimal issues, proving that Rust is a stable and efficient

> language for their projects.

Reviewer: ```suggestion
language for such projects.
```
